apps/roam/src/utils/initializeObserversAndListeners.ts

```diff
@@ -287,6 +292,52 @@ export const initObservers = async ({
     }
   };
 
+  const nodeTagHoverListener = (e: Event) => {
```

Comment:
As discussed in our last meeting, this should be an html observer instead of a hover listener.  This will make it fire far less.

Reply:
yeah my bad, i asked for review before our meeting then we had the review in meeting and i did not make this pr a draft pr after meeting